tests/test_belgium_calendar.py

```diff
@@ -0,0 +1,123 @@
+        expected_sessions = [
+            # In 2010, Labor Day fell on a Saturday, so the market should be
+            # open on both the prior Friday and the following Monday.
+            pd.Timestamp('2010-04-30', tz='UTC'),
+            pd.Timestamp('2010-05-03', tz='UTC'),
+            # Christmas also fell on a Saturday, meaning Boxing Day fell on a
+            # Sunday. The market should still be open on both the prior Friday
+            # and the following Monday.
+            pd.Timestamp('2010-12-24', tz='UTC'),
+            pd.Timestamp('2010-12-27', tz='UTC'),
+        ]
+
+        for session_label in expected_sessions:
+            self.assertIn(session_label, self.calendar.all_sessions)
+
+    def test_half_days(self):
+        half_days = [
+            # In 2010, Christmas Eve and NYE are on Friday, so they should be
+            # half days.
+            pd.Timestamp('2010-12-24', tz='Europe/Brussels'),
+            pd.Timestamp('2010-12-31', tz='Europe/Brussels'),
+        ]
+        not_half_days = [
+            # In Dec 2011, Christmas Eve and NYE were both on a Saturday, so
+            # the preceding Fridays should be full days.
+            pd.Timestamp('2011-12-23', tz='Europe/Brussels'),
+            pd.Timestamp('2011-12-30', tz='Europe/Brussels'),
+        ]
+
+        for half_day in half_days:
+            half_day_close_time = self.calendar.next_close(half_day)
+            self.assertEqual(
+                half_day_close_time,
+                # Early close is 2:05 PM.
+                half_day + pd.Timedelta(hours=14, minutes=5),
+            )
+        for full_day in not_half_days:
+            full_day_close_time = self.calendar.next_close(full_day)
+            self.assertEqual(
+                full_day_close_time,
+                # Normal close is 5:30 PM.
+                full_day + pd.Timedelta(hours=17, minutes=30),
+            )
+
+    def test_old_holidays(self):
+        """
+        Test the before and after of holidays that are no longer observed.
+        """
+        all_sessions = self.calendar.all_sessions
+
+        expected_holidays = [
+            pd.Timestamp('2000-06-01', tz='UTC'),  # Ascension Day
+            pd.Timestamp('2001-06-04', tz='UTC'),  # Whit Monday
+            pd.Timestamp('2000-07-21', tz='UTC'),  # Independence Day
+            pd.Timestamp('2001-12-31', tz='UTC'),  # New Year's Eve
+        ]
+
+        for session_label in expected_holidays:
+            self.assertNotIn(session_label, all_sessions)
+
+        expected_sessions = [
+            pd.Timestamp('2001-05-24', tz='UTC'),  # Ascension Day
+            pd.Timestamp('2002-05-20', tz='UTC'),  # Whit Monday
+            pd.Timestamp('2003-07-21', tz='UTC'),  # Independence Day
+            pd.Timestamp('2002-12-31', tz='UTC'),  # New Year's Eve
+        ]
+
+        for session_label in expected_sessions:
+            self.assertIn(session_label, all_sessions)
+
+    def test_start_end(self):
```

Comment:
I moved this test into `ExchangeCalendarTestBase` in the branch I recently merged, so you should be able to remove this test (and the corresponding tests for the other countries) after rebasing